packages/now-python/test/fixtures/14-unicode-handler/now.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "*.py",
+      "use": "@now/python"
+    }
+  ],
+  "probes": [
+    {
+      "path": "/",
+      "method": "POST",
+      "body": {
+        "name": "Χριστοφορε"
+      },
+      "status": 200
```

Comment:
Thank for the PR 🙌

I believed you can use `"mustContain": "hello, Χριστοφορε"` inside `probes`.

Other than that, it's looking good to me.

Reply:
I had tried that, but this error occurred because it's still in a decoded state: 

![image](https://user-images.githubusercontent.com/4681330/65879577-175e5580-e35e-11e9-991e-cfe986e7f154.png)

I can test the escape sequence string instead if you would prefer. LMK.

Reply:
That's strange. Does the same thing happen with `fetch(deploymentUrl)` in probe.js?

Reply:
No, probe.js is fine.